packages/server/src/api/notifRoutes.ts

```diff
@@ -0,0 +1,112 @@
+import { ServerRoute, ResponseObject } from "@hapi/hapi";
+import { NotifBody } from "@template/common";
+import { NotifModel } from "../entity/NotifModel";
+import { NotifPayload } from "../joi";
+import * as dotenv from "dotenv";
+import * as webPush from "web-push";
+import { DeepPartial } from "typeorm";
+
+// configure env vars for VAPID
+dotenv.config();
+
+// if env vars not found, then throw an error early
+if (!process.env.EMAIL || !process.env.PUBLICKEY || !process.env.PRIVATEKEY) {
+  throw new Error(
+    "please add a .env file with keys+email in packages/server. ask alex/eddy for deets."
+  );
+} else {
+  webPush.setVapidDetails(
+    process.env.EMAIL,
+    process.env.PUBLICKEY,
+    process.env.PRIVATEKEY
+  );
+}
+
+export const notifRoutes: ServerRoute[] = [
```

Comment:
Write tests plzz

Reply:
smh i left this as a previous review comment so you wouldnt have to